.github/workflows/detekt.yml

```diff
@@ -0,0 +1,62 @@
+# This is a basic workflow that is manually triggered
+
+name: Detekt 
+
+# Controls when the action will run. Workflow runs when manually triggered using the UI
+# or API.
+on:
+  workflow_dispatch:
+    # Inputs the workflow accepts.
+    inputs:
+      name:
+        # Friendly description to be shown in the UI instead of 'name'
+        description: 'Detekt'
+        # Default value if no value is explicitly provided
+        default: 'World'
+        # Input has to be provided for the workflow to run
+        required: true
+        # The data type of the input
+        type: string
+  
+  push:
+    branches:
+      - feature/detekt
+  pull_request:
+    branches:
+      - "master" 
```

Comment:
Probably we want to run Detekt in every PR we do, not only in master (for which we cannot create PR), so this needs to be reviewed as well

Reply:
Same as before. Intended and provisional.